streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -138,14 +136,14 @@ public void close(final boolean clean,
             return;
         }
         log.debug("Closing");
-        boolean committedSuccessfully = false;
+        boolean committed = false;
         try {
             if (clean) {
                 commit();
-                committedSuccessfully = true;
+                committed = true;
             }
         } finally {
-            closeStateManager(committedSuccessfully);
+            closeStateManager(clean && committed);
```

Comment:
Why do we need to include `clean` here? (Was this another bug?) I thought for standbys this does not matter?

Reply:
Good point, since standby tasks do not have eos anyways today.